src/k8s/pkg/utils/file.go

```diff
@@ -138,3 +125,38 @@ func ValueInSlice[T comparable](key T, list []T) bool {
 
 	return false
 }
+
+var ErrUnknownMount = errors.New("mount is unknown")
```

Comment:
do we care to export the error here?

Reply:
People and tests might want to check it.